### DISCUSSION TITLE
Example of GEBCO data usage with GMT

### DISCUSSION THREAD

**Discussion by meerfrau** [GenericMappingTools/gmt#7451] [Documentation development discussion]
Hi,

I am new to GMT and would be interested in any example GMT file using current [GEBCO data](https://www.gebco.net/data_and_products/gridded_bathymetry_data/gebco_2023/).

**Comment by PaulWessel** [comment]
Please pose such questions on the GMT [forum](https://forum.generic-mapping-tools.org) and not the GitHub which is for the Developers to discuss things.

A quick example of GEBCO dumbed down to 30 arc minutes and plotted in the Hammer projection:

`gmt grdimage @earth_gebco_30m -JH15c -B -png map`
